playground/examples/react/yjs/extension.ts

```diff
@@ -0,0 +1,55 @@
+import { defineBasicExtension } from 'prosekit/basic'
+import {
+  defineBaseCommands,
+  defineBaseKeymap,
+  defineDoc,
+  defineParagraph,
+  defineText,
+  union,
+} from 'prosekit/core'
+import { defineBlockquote } from 'prosekit/extensions/blockquote'
+import { defineBold } from 'prosekit/extensions/bold'
+import { defineCode } from 'prosekit/extensions/code'
+import { defineDropCursor } from 'prosekit/extensions/drop-cursor'
+import { defineHeading } from 'prosekit/extensions/heading'
+import { defineImage } from 'prosekit/extensions/image'
+import { defineItalic } from 'prosekit/extensions/italic'
+import { defineLink } from 'prosekit/extensions/link'
+import { defineList } from 'prosekit/extensions/list'
+import { defineModClickPrevention } from 'prosekit/extensions/mod-click-prevention'
+import { defineStrike } from 'prosekit/extensions/strike'
+import { defineTable } from 'prosekit/extensions/table'
+import { defineUnderline } from 'prosekit/extensions/underline'
+import { defineVirtualSelection } from 'prosekit/extensions/virtual-selection'
+import { defineYjs } from 'prosekit/extensions/yjs'
+import { Awareness } from 'y-protocols/awareness'
+import * as Y from 'yjs'
+
+export function defineExtension(doc?: Y.Doc, awareness?: Awareness) {
+  return doc && awareness
```

Comment:
As an example, maybe we should just enforce `defineExtension` to receive `doc` and `awareness` here?